Sources/Wrap.swift

```diff
@@ -546,6 +547,27 @@ private extension Wrapper {
     }
 }
 
+// MARK: - Nil Handling
+
+private protocol WrapNillable {
```

Comment:
Nice solution! How about calling this `WrapOptional` instead, to match `Optional`?